pep-0727.rst

```diff
@@ -0,0 +1,551 @@
+PEP: 727
+Title: Documentation Metadata in Typing
+Author: Sebastián Ramírez <tiangolo@gmail.com>
+Sponsor: Jelle Zijlstra <jelle.zijlstra@gmail.com>
```

Comment:
Just to formally confirm it, I agree with being the sponsor.

Reply:
Thanks a lot for that! 🙇